tests/e2e/kepler_test.go

```diff
@@ -131,7 +133,8 @@ func TestTaint_WithToleration(t *testing.T) {
 	assert.NoError(t, err, "failed to taint node %s", node)
 	t.Cleanup(func() {
 		// remove taint
-		f.TaintNode(node, fmt.Sprintf("%s-", e2eTestTaint.Key))
+		err := f.TaintNode(node, fmt.Sprintf("%s-", e2eTestTaint.Key))
+		assert.NoError(t, err, "could not remove taint from node")
 	})
```

Comment:
Thanks a lot for the fix above.. can we do the same here 🙏 ?

Reply:
@vprashar2929 may be in a followup PR?

Reply:
Completely forgot this😅 I will fix this as well in the PR